src/auxiliary/math.jl

```diff
@@ -180,6 +192,18 @@ multiplication.
     end
 end
 
+# TODO: Performance tuning for `inv_ln_mean` (Float32), please find below
```

Comment:
```suggestion
# TODO: This may need some performance tuning, e.g., to optimize the
#       cutoff value `epsilon_f2` or the number of terms in the Taylor expansion
```